src/styles/card.scss

```
@@ -0,0 +1,88 @@
+body {
+  margin: 0;
+  padding: 0;
+  font-size: 10px;
+  font-weight: 400;
+}
+
+.product-card {
+  width: 200px;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+
+  &__img {
+    height: 134px;
+    width: 160px;
+    display: block;
+    margin: 0 auto 40px;
+  }
+
+  &__title {
+    margin-bottom: 4px;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: $main-color;
+  }
+
+  &__sku {
+    margin: 0 0 16px;
+    line-height: 14px;
+    color: $alt-color;
+  }
+
+  &__rating {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 24px;
+  }
+
+  &__reviews {
+    line-height: 14px;
+    color: $main-color;
+  }
+
+  &__pricing {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 16px;
+    line-height: 18px;
+
+    &-label {
+      font-size: 12px;
+      color: $alt-color;
+    }
+
+    &-amount {
+      font-size: 16px;
+      font-weight: 700;
+      color: $main-color;
+    }
+  }
+
+  &__button {
+    box-sizing: border-box;
+    display: block;
+    width: 100%;
+    height: 40px;
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 40px;
+    text-align: center;
+    text-decoration: none;
+    text-transform: uppercase;
+    background-color: $butt-color;
+    border-radius: 5px;
+    border: 1px solid $butt-color;
+    color: #fff;
+
+    &:hover {
+      color: $butt-color;
```

`butt-color`? Use full name for it, or another variant. For example:
```suggestion
      color: $button-color;
```
If another programmer comes in and doesn't understand it like I do, and Googles it, he'll be surprised and shocked at the same time.